book/website/collaboration/inclusive-events/inclusive-events-schedules.md

```diff
@@ -0,0 +1,286 @@
+
+Consider designing sessions that support multiple channels of engagement to be as inclusive of as many accessibility requirements and preferences as possible. 
+Because all events are limited by the resources available to plan and host the event, it is important to provide clear guidance on what engagement means for your event and to work with a representative with lived experience to identify what methods of engagement your event will need to support.
+
+Ensure that you provide an outline of your guidance in both a written format before the event (preferably in plain text), and also include an explanation in your event introduction of what engagement is expected, as well as what pathways are available for contributions.
+
+(cl-inclusive-events-defining-goals)=
+### Sharing the Goals of your Event
+Consider outlining and sharing a set of goals or planned outputs from your event with your audience when you begin a session. 
+For example, a workshop may present learning objectives or an example of the planned finished product. 
+
+By clearly articulating the event's objectives, you will be able to create a sense of transparency and purpose for the session. 
+This clarity allows all attendees, including people with lived experience, to understand what to expect from the session and reflexively evaluate whether their participation aligns with those goals. 
+
+Additionally, sharing an event's goals can foster a sense of trust among attendees by presenting how the perspectives and contributions of people with lived experience are valued by the research project team. 
+This can be especially important for "data-driven" research that otherwise places a large focus on algorithmic techniques. 
+
+Inclusive research requires not only active participation from people with lived experience and diverse backgrounds, but also actively ensuring that their opinions and perspectives are heard and able to influence the research directly. 
+By orientating your attendees around a goal, you can provide a shared understanding that will support meaningful and inclusive engagement in your event.
+
+(cl-inclusive-events-inclusive-content)=
+### Planning for Inclusion
+As part of planning the program, consider how you will support and facilitate relationship-building and the active inclusion of people with lived experience in your event's sessions.
+
+Research events can often be focused on actively engaging researchers who have a clear pathway to engagement and involvement in an event, for example explaining their research or asking questions of colleagues to better understand what research methods led to a new discovery. 
+If your event plans to involve people with lived experience, you should consider designing your event's sessions to allow for experts through experience to engage in the material equitably alongside any attendees who are experts by profession, instead of being segregated out for being "non-researchers". 
```

Comment:
Oof yes. 
Maybe also add something on how to ensure that both these perspectives are considered as equal and the researcher perspective should not be leading? I feel like this should be stressed more (based on some experiences from the same session last week...)